project-base/app/src/DataFixtures/Demo/ParameterDataFixture.php

```diff
@@ -158,8 +158,8 @@ public function load(ObjectManager $manager)
         $product2 = $this->getReference(ProductDataFixture::PRODUCT_PREFIX . '2');
 
         foreach ($this->domain->getAllLocales() as $locale) {
-            $parameterValueRed = $this->getParameterValue($locale, t('black', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $locale), '#000000');
-            $this->addParameterValueToProduct($product2, $parameterColor, $parameterValueRed);
+            $parameterValueBlack = $this->getParameterValue($locale, t('black', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $locale), '#000000');
+            $this->addParameterValueToProduct($product2, $parameterColor, $parameterValueBlack);
             $parameterValueMetal = $this->getParameterValue($locale, t('metal', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $locale));
```

Comment:
in the following two foreaches, there are also bad names for the variables - there is `$parameterValueMetal` while it should be `$parameterValuePlastic` and `$parameterValueWood`, could you please fix that as well? :pray: